src/auto/settings.rs

```diff
@@ -77,6 +77,10 @@ pub trait SettingsExt: 'static {
         gtk_application_prefer_dark_theme: bool,
     );
 
+    fn get_property_gtk_cursor_aspect_ratio(&self) -> f32;
```

Comment:
No `feature = "v3_24_XX"`?

Reply:
This a) needs annotations in the C code, and b) configured in the `Gir.toml`

Reply:
It was added in Gtk 3.24.11 https://gitlab.gnome.org/GNOME/gtk/-/blob/gtk-3-24/NEWS#L90

Reply:
Can you (generally) also link to the upstream MR (or issues) you created for these things?